src/app/repositories/userRepository.ts

```diff
@@ -0,0 +1,9 @@
+import { httpClient, queryKeyMap } from '@libs/index';
+
+export const userRepository = {
+  async signin({ email, password }: { email: string; password: string }) {
+    return await httpClient.post('/users/sign-in', { email, password });
```

Comment:
```suggestion
    return httpClient.post('/users/sign-in', { email, password });
```
이렇게해도 잘 동작합니다.
자바스크립트에서 return은 await를 기다려줌

Reply:
아하 넵 수정했습니다.